libraries/SITL/SIM_Sailboat.h

```diff
@@ -0,0 +1,56 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  sailboat simulator class
+*/
+
+#pragma once
+
+#include "SIM_Aircraft.h"
+
+namespace SITL {
+
+/*
+  a sailboat simulator
+ */
+class Sailboat : public Aircraft {
+public:
+    Sailboat(const char *home_str, const char *frame_str);
+
+    /* update model by one time step */
+    void update(const struct sitl_input &input);
+
+    /* static object creator */
+    static Aircraft *create(const char *home_str, const char *frame_str) {
+        return new Sailboat(home_str, frame_str);
+    }
+
+private:
+
+    void calc_lift_and_drag(float wind_speed, float angle_of_attack_deg, float& lift, float& drag);
```

Comment:
These functions should all have the comments from the C++ implementation by them. (In fact it's more important to have them here then in the cpp file).

Reply:
ok, comments copied over from the .cpp file to the .h file.